app/assets/javascripts/application.js

```diff
@@ -11,6 +11,7 @@
 // about supported directives.
 //
 //= require jquery
+//= require bootstrap-sprockets
```

Comment:
Just saying because we are not using sprockets afaik, and this has some consequences when building the RPM. One easy step would be to replace this line with:

```javascript
//= require bootstrap
```